panels/class-debug-bar-panel.php

```diff
@@ -60,4 +60,42 @@ function title( $title = null ) {
 	function debug_bar_classes( $classes ) {
 		return $classes;
 	}
+
+	/**
+	 * Render a (floating) block with a title and content.
+	 *
+	 * Generally these blocks are used at the top of a panel for a quick info overview.
+	 *
+	 * {@internal The list of allowed HTML tags is based on the most common HTML tags used
+	 *            by this plugin and its add-ons.}}
+	 *
+	 * @since 0.10.0
+	 *
+	 * @param string $title   The title of the block.
+	 * @param mixed  $content The value/other content for the block.
+	 *
+	 * @return string|void
+	 */
+	protected function render_panel_info_block( $title, $content ) {
+		if ( is_int( $content ) ) {
```

Comment:
Should this be `is_numeric()`? If it's an int we probably wouldn't have to sanitize it